<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lysstyring</class>
 <widget class="QDialog" name="Lysstyring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>14</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lys styring</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="16" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="slutlabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Slut tid:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QTimeEdit" name="startTimeEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>190</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QTimeEdit" name="slutTimeEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>190</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QPushButton" name="OnOffButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1677721</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#OnOffButton{
border-image: url(:l2off.png);


}
#OnOffButton:checked
{
border-image: url(:l2on.png);
}
#OnOffButton:enabled
{
	color: red
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="gemPushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Gem</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="startlabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Start tid:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Tryk for manual styring:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sluk automatisk lys mellem:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="SensorLabel">
     <property name="text">
      <string>Sensor:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QLabel" name="statuslabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>49</width>
       <height>29</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{
border-image: url(:offt.png);


}
#pushButton:checked
{
border-image: url(:ont.png);
}
#OnOffButton:enabled
{
	color: red
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
